### PR TITLE
Update virus-scanner-support.adoc

### PR DESCRIPTION
**Pull request by hodyroff**
@phil-davis please double-check for correctness with oC10 server

Backport to 10.11 and 10.10

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc [hunk 1]
```diff
@@ -334,6 +334,7 @@ During an upload these actions are taken:
 * The upload is blocked.
 * The event is logged in the owncloud server log.
 * The event is reported and/or logged by the client / Web UI.
+* The affected file will be deleted on the desktop by the desktop client and therefor be lost completely, as the desktop client does not support a local trashbin.
```
Comment (EParzefall) [inline modules/admin_manual/pages/configuration/server/virus-scanner-support.adoc:337]:
```suggestion
* The affected file will be deleted on the desktop by the desktop client and therefore be lost completely, as the desktop client does not support a local trash bin.
```

**Review by EParzefall — APPROVED** [review]

**Comment by mmattel** [comment]
@phil-davis can you have a look if this is technically correct? @EParzefall will do based on your ok the merge and the necessary backports.

**Comment by phil-davis** [comment]
> @phil-davis can you have a look if this is technically correct? @EParzefall will do based on your ok the merge and the necessary backports.

If I try to upload the EICAR test "virus" file on the classic webUI, then it correctly fails to upload and I get a message about the file containing a virus - good. And in that scenario the file never gets to the server, and so never syncs to the desktop client, and so the desktop client never had the file anyway (so doesn't "delete" it)

If I put the file into the local files on my desktop system, the desktop client tries to sync it up to the sever, and it correctly reports that a virus was detected in the file - good. But the desktop client does not delete the local copy of the "virus" file.

I am using Desktop Client 2.11.1 (build 8438) Aug 31 2022 10:28:50 on Ubuntu 20;04.

So I don't see the "Desktop Client deleted the local virus file" behavior.

**Comment by mmattel** [comment]
> So I don't see the "Desktop Client deleted the local virus file" behavior.

Closing therefore

**Pull request closed without merging**